mod/metrics/set/type/metric_answer/listings.rb

```diff
@@ -34,6 +41,16 @@
     end
   end
 
+  def citations_count_badge
+    wrap_with :span, card.source_card&.item_names&.size, class: "badge badge-light border"
```

Comment:
doesn't 
```suggestion
    count_badge :source
```
work here?

Reply:
...hmm, maybe that will use the name "sources" instead of citations...